apps/client/src/components/leaderboard/leaderboard.module.css

```
@@ -5,3 +5,27 @@
   padding: 0.5em;
   gap: 2em;
 }
+.goldBorderColor {
+  border-color: var(---gold-leaderboard-color);
+}
+.silverBorderColor {
+  border-color: var(---silver-leaderboard-color);
+}
+.bornzeBorderColor {
+  border-color: var(---bronze-leaderboard-color);
+}
+.dividerGold {
+  height: 100%;
+  width: 0.1em;
+  background-color: var(---gold-leaderboard-color);
+}
+.dividerSilver {
+  height: 100%;
+  width: 0.1em;
+  background-color: var(---silver-leaderboard-color);
+}
+.dividerBronze {
+  height: 100%;
+  width: 0.1em;
+  background-color: var(---bronze-leaderboard-color);
```

I think it's better to create one class for divider, and a class for each color, and then use them both accordingly.

```suggestion
.divider {
  height: 100%;
  width: 0.1em;
}
.goldBackground{
  background-color: var(---gold-leaderboard-color);
}
{/*...*/}